test/functional/rpc_packages.py

```diff
@@ -369,23 +370,65 @@ def test_maxfeerate_maxburn_submitpackage(self):
         minrate_btc_kvb = min([chained_txn["fee"] / chained_txn["tx"].get_vsize() * 1000 for chained_txn in chained_txns])
         chain_hex = [t["hex"] for t in chained_txns]
         pkg_result = node.submitpackage(chain_hex, maxfeerate=minrate_btc_kvb - Decimal("0.00000001"))
+
+        # First tx failed in single transaction evaluation, so package message is generic
+        assert_equal(pkg_result["package_msg"], "transaction failed")
         assert_equal(pkg_result["tx-results"][chained_txns[0]["wtxid"]]["error"], "max feerate exceeded")
         assert_equal(pkg_result["tx-results"][chained_txns[1]["wtxid"]]["error"], "bad-txns-inputs-missingorspent")
         assert_equal(node.getrawmempool(), [])
 
+        # Make chain of two transactions where parent doesn't make minfee threshold
+        # but child is too high fee
+        # Lower mempool limit to make it easier to fill_mempool
+        self.restart_node(0, extra_args=[
+            "-datacarriersize=100000",
+            "-maxmempool=5",
+        ])
+
+        fill_mempool(self, node, self.wallet)
+        while node.getrawmempool() != []:
+            self.generate(node, 1)
```

Comment:
Note that this can change the mempool min fee

Reply:
mm yeah let me make this a bit more bullet proof

Reply:
Well I don't think you need get rid of the transactions here - you just want to make sure they get cleared by the end of the test right?

Reply:
felt like I had a reason for this, but apparently not. At end of test I empty mempool on node restart, so removed